Sources/REPRLRun/main.swift

```diff
@@ -37,7 +37,12 @@ if ctx == nil {
 }
 
 let argv = convertToCArray(Array(CommandLine.arguments[1...]))
+#if os(macOS)
+// Set the DYLD_FRAMEWORK_PATH env variable for the shell to run
```

Comment:
This also feels misplaced here. Maybe we could just forward our environment to the child instead? Then you'd have to `DYLD_FRAMEWORK_PATH=xyz ./REPRLRun path/to/jsc` which would be ok I guess

Reply:
Yes it should be fine, but I just find that with the static build of JavascriptCore of libJavascriptCore.a, we now do not need the DYLD_FRAMEWORK_PATH for the shell to run, so we can discard this commit at all if we properly solve the static build (Indeed a mess especially for some version of WebKit code fro some code between release tag). 

Also for the first problem